jcl/src/java.base/share/classes/java/lang/Throwable.java

```diff
@@ -486,7 +494,11 @@ private void readObject(ObjectInputStream s)
 			} else {
 				/*[MSG "K0C00", "Non-standard List class not permitted in suppressedExceptions serial stream"]*/
 				throw new java.io.StreamCorruptedException(com.ibm.oti.util.Msg.getString("K0C00")); //$NON-NLS-1$
+/*[IF JAVA_SPEC_VERSION >= 9]*/
```

Comment:
What's the point of this? The code for both cases is the same, a terminating `}`.

Reply:
This avoids confusing editors that don't understand the preprocessor comments: now the braces are balanced in both the raw source and (still) the processed source.